frappe/email/receive.py

```diff
@@ -900,43 +900,3 @@ def as_dict(self):
 			"has_attachment": 1 if self.attachments else 0,
 			"seen": self.seen_status or 0,
 		}
-
-
-class TimerMixin:
-	def __init__(self, *args, **kwargs):
-		self.timeout = kwargs.pop("timeout", 0.0)
-		self.elapsed_time = 0.0
-		self._super.__init__(self, *args, **kwargs)
-		if self.timeout:
-			# set per operation timeout to one-fifth of total pop timeout
-			self.sock.settimeout(self.timeout / 5.0)
-
-	def _getline(self, *args, **kwargs):
-		start_time = time.monotonic()
-		ret = self._super._getline(self, *args, **kwargs)
-
-		self.elapsed_time += time.monotonic() - start_time
-		if self.timeout and self.elapsed_time > self.timeout:
-			raise EmailTimeoutError
```

Comment:
This custom exception is used in places, it should be replaced with standard exception now.